File: Materias Obrigatorias/2º Semestre/AED I - Algoritmos e Estruturas de Dados I/Digiampietri/Códigos/EstruturaSimples.java
public class EstruturaSimples {
  public static final int alturaMaxima = 225;

  public static void main(String[] args) {
    pesoAltura pessoa1 = new pesoAltura();
	pessoa1.peso = 80;
	pessoa1.altura = 185;
    System.out.print("Peso: " + pessoa1.peso + ", Altura "+pessoa1.altura+". ");
	if (pessoa1.altura>alturaMaxima) {
	  System.out.println("Altura acima da maxima.\n");
	}
	else System.out.println("Altura abaixo da maxima.\n");
  }
}

File: Materias Obrigatorias/2º Semestre/AED I - Algoritmos e Estruturas de Dados I/Digiampietri/Códigos/EstruturaSimples.c
#include <stdio.h>
#define alturaMaxima 225

typedef struct
{
  int peso; // peso em quilogramas
  int altura; // altura em centimetros
} pesoAltura;

int main() {
  pesoAltura pessoa1;
  pessoa1.peso = 80;
  pessoa1.altura = 185;

  printf("Peso: %d, Altura %d. ", pessoa1.peso, pessoa1.altura);
  if (pessoa1.altura>alturaMaxima) {
    printf("Altura acima da maxima.\n");
  }
  else printf("Altura abaixo da maxima.\n");
 
  return 0;
}
